### PR TITLE
[FIX] base_kanban_stage: following on #949

### PR DESCRIPTION
**Pull request by tafaRU**
Following on https://github.com/OCA/server-tools/pull/949

### CODE REVIEW

**Comment by tafaRU** [comment]
Despite reviews on https://github.com/OCA/server-tools/pull/949 we missed this further change.
Please merge it asap.

**Review by eLBati — APPROVED** [review]

File: base_kanban_stage/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     'name': 'Kanban - Stage Support',
     'summary': 'Provides stage model and abstract logic for inheritance',
-    'version': '10.0.1.0.1',
+    'version': '10.0.1.0.2',
```
Comment (lasley) [inline base_kanban_stage/__manifest__.py]: Because this will require an update, please raise to `10.0.1.1.0`

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @tafaRU - minor change then I'll fast-track

**Comment by tafaRU** [comment]
@lasley, done!

**Comment by lasley** [comment]
Fast tracked to resolve obvious production issue. Thanks @tafaRU

**Comment by tafaRU** [comment]
@lasley, also https://github.com/OCA/server-tools/pull/968 could be merged.

**Pull request merged**